apps/CA.pl.in

```diff
@@ -77,6 +91,19 @@ sub copy_pemfile
     return $found == 2 ? 0 : 1;
 }
 
+sub copy_any
```

Comment:
Quite frankly, I'd rather see File::Copy being used.  It's been with perl all the way since 5.002, so it's safe to use.